buildenv/jenkins/common/build.groovy

```diff
@@ -155,8 +155,15 @@ def checkout_pullrequest() {
 
         // Setup PR_IDs for dependent changes
         for (DEPEND in DEPENDS_ARRAY) {
-            String REPO = DEPEND.substring(DEPEND.indexOf("/") + 1, DEPEND.indexOf("#"));
-            String PR_ID = DEPEND.substring(DEPEND.indexOf("#") + 1, DEPEND.length());
+            if (DEPEND.contains('http')) {
+                // eg htts://github.com/elclipse/omr/pull/123
+                def urlList = DEPEND.tokenize('/')
+                REPO = urlList[3]
+                PR_ID = urlList[5]
+            } else {
+                String REPO = DEPEND.substring(DEPEND.indexOf("/") + 1, DEPEND.indexOf("#"));
```

Comment:
I expect this will fail if it doesn't contain `#`.

Reply:
But it should contain. Is there another use case you are thinking of?

Reply:
If someone says 'depends org/repo$123' (where they meant '#' instead of '$'), this should behave gracefully.